core/dbt/adapters/base/impl.py

```diff
@@ -781,14 +782,49 @@ def get_catalog(self, manifest):
         Returns an agate.Table of catalog information.
         """
         # make it a list so macros can index into it.
-        context = {'databases': list(manifest.get_used_databases())}
+        context = {
+            'databases': list(manifest.get_used_databases()),
+            'information_schema': self.get_information_schema_mapping(manifest)
+        }
+
         table = self.execute_macro(GET_CATALOG_MACRO_NAME,
                                    context_override=context,
                                    release=True)
 
         results = self._catalog_filter_table(table, manifest)
         return results
 
+    @available
```

Comment:
I don't think you really want this available in macros/materializations, given that you're already injecting its result into the context.